releases/release-6.5.5.md

```diff
@@ -0,0 +1,87 @@
+---
+title: TiDB 6.5.5 Release Notes
+summary: 了解 TiDB 6.5.5 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.5 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.5
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.5#version-list)
+
+## 改进提升
+
++ TiDB **tw@qiancai**
+
+    - 支持从高版本回退到 6.5 [#45570](https://github.com/pingcap/tidb/issues/45570) @[wjhuang2016](https://github.com/wjhuang2016)
+    - (dup): release-7.3.0.md > # 稳定性 * 新增部分优化器提示 [#45520](https://github.com/pingcap/tidb/issues/45520) @[qw4990](https://github.com/qw4990)
+    - 添加 coprocessor 相关 request source 信息 [#46514](https://github.com/pingcap/tidb/issues/46514) @[you06](https://github.com/you06)
+
++ TiKV **tw@qiancai**
+
+    - 增加 pd-client中连接重试过程中backoff的功能，减小PD压力 [#15428](https://github.com/tikv/tikv/issues/15428) @[nolouch](https://github.com/nolouch)
+    - Titan：避免写Titan manifest文件时持有锁 [#15351](https://github.com/tikv/tikv/issues/15351) @[Connor1996](https://github.com/Connor1996)
+    - 压缩check_leader请求. [#14839](https://github.com/tikv/tikv/issues/14839) @[you06](https://github.com/you06)
+    - 增加snapshot相关的监控 [#15401](https://github.com/tikv/tikv/issues/15401) @[SpadeA-Tang](https://github.com/SpadeA-Tang)
```

Comment:
压缩 check_leader 请求，节约 resolved-ts 组件所产生的流量

Reply:
remove the following because this issue has also been fixed by https://github.com/tikv/tikv/issues/14553 in v6.5.4 https://docs.pingcap.com/zh/tidb/dev/release-6.5.4#%E6%94%B9%E8%BF%9B%E6%8F%90%E5%8D%87

- 压缩check_leader请求. [#14839](https://github.com/tikv/tikv/issues/14839) @[you06](https://github.com/you06)